models/cam-se/model_mod.f90

```diff
@@ -1305,7 +1330,15 @@ subroutine get_se_quad_vals(state_handle, ens_size, varid, obs_qty, corners, &
          call get_se_values_from_varid(state_handle,  ens_size, corners(icorner), & 
                                     level_one_array, varid, quad_vals(icorner,:),my_status)
 
-         if (any(my_status /= 0)) return
+         if (any(my_status /= 0)) then
+            ! Debug se_values
+            if(debug_level > 12) then
+               write(string1,*)'get_se_values_from_varid failed varid, corner,  my_status(:) = ', &
+                                varid, corners(icorner),my_status
+               call error_handler(E_MSG,routine,string1,source,revision,revdate)
+            endif
+            return
+         endif
```

Comment:
same message comment: do you need to have task 0 print this?
You could set a conditional debug in a debugger to catch these on any task if you need to debug.